modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java

```diff
@@ -759,6 +760,24 @@ public Response addMediaPackage(@Context HttpServletRequest request, @PathParam(
       int episodeDCCatalogNumber = 0;
       boolean hasMedia = false;
       if (ServletFileUpload.isMultipartContent(request)) {
+        // Validate first
+        for (FileItemIterator iter = new ServletFileUpload().getItemIterator(request); iter.hasNext();) {
+          FileItemStream item = iter.next();
+          if (item.isFormField()) {
+            String fieldName = item.getFieldName();
+            String value = Streams.asString(item.openStream(), "UTF-8");
+            if (dcterms.contains(fieldName)) {
+              if ("created".equals(fieldName) || "date".equals(fieldName) || "temporal".equals(fieldName)) {
```

Comment:
The Dublin Core properties are defined in that interface:
https://github.com/opencast/opencast/blob/develop/modules/dublincore/src/main/java/org/opencastproject/metadata/dublincore/DublinCore.java